src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: $card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: 0 48px 46px 0;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__logo {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    object-fit: cover;
+    max-width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: $medium-text;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $primary-grey;
+    margin-bottom: 16px;
+  }
+
+  &__response {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-item {
+      color: $primary-grey;
+    }
```

Comment:
`-item` is not an element or modifier of `card__price`, so it's a wrong usage of nesting. Although it provides ability to store some class name (like variable), we shouldn't use it for class name combining (it's not clear if it's nested class, or just other class with name that includes this word)